crates/hulk_behavior_simulator/src/game_controller.rs

```diff
@@ -90,6 +104,16 @@ fn game_controller_controller(
         GameState::Playing => {}
         GameState::Finished => {}
     }
+
+    if let (Some(_), Team::Opponent) = (
```

Comment:
Why only do this if the opponent is kicking team?

Reply:
Oversight, fixed